priv/graphql/schemas/admin/admin_schema.gql

```diff
@@ -26,5 +28,7 @@ type AdminMutation @protected{
   account: AccountAdminMutation
   "Modify domains properties or create/destroy domain"
   domains: DomainAdminMutation
+  "Session management"
```

Comment:
It would be great to use either noun or verb form consistently, i.e. either "Manage sessions", "Modify domains" or "Account management", "Domain management", same for queries. Maybe the noun form would be better for categories?